MSAL/src/requests/MSALRequestParameters.h

```diff
@@ -69,4 +71,6 @@
 #pragma mark Methods
 - (void)setScopesFromArray:(NSArray<NSString *> *)array;
 - (void)setCloudAuthorityWithCloudHostName:(NSString *)cloudHostName;
+- (void)setClaims:(NSString *)claims NS_UNAVAILABLE;
+- (BOOL)setClaims:(NSString *)claims error:(NSError **)error;
```

Comment:
Can this live in MSID space instead?

Reply:
Talked Olga offline, probably we need to have one request parameter class in MSID space such that the duplicate logic could be unified :)